pkg/crc/preflight/preflight.go

```diff
@@ -30,6 +77,8 @@ type Check struct {
 	flags              Flags
 	cleanupDescription string
 	cleanup            CleanUpFunc
+
+	labels labels
```

Comment:
Looks good! Maybe convert `flags` to a label also?

Reply:
Yeah, that's next, I started here https://github.com/cfergeau/crc/commit/256eae0462dc1708cc7f0485289d1570ef5189f8 but I prefer to get this initial PR in first, and then improve on it.